x-pack/plugins/enterprise_search/public/applications/app_search/components/meta_engine_creation/constants.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+export const DEFAULT_LANGUAGE = 'Universal';
+
+export const META_ENGINE_CREATION_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.metaEngineCreation.title',
+  {
+    defaultMessage: 'Create a meta engine',
+  }
+);
+
+export const META_ENGINE_CREATION_FORM_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.metaEngineCreation.form.title',
+  {
+    defaultMessage: 'Name your meta engine',
+  }
+);
+
+export const META_ENGINE_CREATION_FORM_SUBMIT_BUTTON_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.metaEngineCreation.form.submitButton.buttonLabel',
+  {
+    defaultMessage: 'Create meta engine',
+  }
+);
+
+export const META_ENGINE_CREATION_FORM_META_ENGINE_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.metaEngineCreation.form.metaEngineDescription',
+  {
+    defaultMessage:
+      'Meta engines allow you to combine multiple engines into one, searchable engine.',
+  }
+);
+
+export const META_ENGINE_CREATION_FORM_DOCUMENTATION_LINK = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.metaEngineCreation.form.documentationLink',
+  {
+    defaultMessage: 'Read the documentation',
+  }
+);
+
+export const META_ENGINE_CREATION_FORM_DOCUMENTATION_DESCRIPTION = (
+  <FormattedMessage
+    id="xpack.enterpriseSearch.appSearch.metaEngineCreation.form.documentationDescription"
+    defaultMessage="{documentationLink} for information about how to get started."
+    values={{
+      documentationLink: (
+        <a
+          href="https://www.elastic.co/guide/en/app-search/current/meta-engines-guide.html"
```

Comment:
(thanks @JasonStoltz for reminding me) We should be using our `DOCS_PREFIX` helper (https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts#L10) for all documentation URLs. This handles generating documentation base URLs which are correctly version-prefixed based on whatever version Kibana is running.

```
```suggestion
          href={`${DOCS_PREFIX}/meta-engines-guide.html`}
```